src/components/theme/Footer/Footer.jsx

```diff
@@ -83,42 +95,23 @@ const Footer = ({ intl }) => {
           />
         </Segment>
         <List horizontal inverted>
-          {/* wrap in div for a11y reasons: listitem role cannot be on the <a> element directly */}
-          <div role="listitem" className="item">
-            <UniversalLink
-              className="item"
-              href={settings.isMultilingual ? `/${lang}/sitemap` : '/sitemap'}
-            >
-              <FormattedMessage id="Site Map" defaultMessage="Site Map" />
-            </UniversalLink>
-          </div>
-          <div role="listitem" className="item">
-            <UniversalLink
-              className="item"
-              href={
-                settings.isMultilingual
-                  ? `/${lang}/accesibility-info`
-                  : '/accesibility-info'
-              }
-            >
-              <FormattedMessage
-                id="Accessibility"
-                defaultMessage="Accessibility"
-              />
-            </UniversalLink>
-          </div>
-          <div role="listitem" className="item">
-            <UniversalLink
-              className="item"
-              href={
-                settings.isMultilingual
-                  ? `/${lang}/contact-form`
-                  : '/contact-form'
-              }
-            >
-              <FormattedMessage id="Contact" defaultMessage="Contact" />
-            </UniversalLink>
-          </div>
+          {siteActions?.map((item) => (
+            <div role="listitem" className="item">
+              <UniversalLink
+                className="item"
+                href={
+                  settings.isMultilingual
+                    ? `/${lang}/${flattenToAppURL(item.url)}`
+                    : flattenToAppURL(item.url)
+                }
+              >
+                {intl.formatMessage(messages.siteActions, {
```

Comment:
@nileshgulia1  I don't see any point of using i18n here. I don't even think it will work.